frontend/src/pages/SkillsMap/StackMap/CourseMapForCurrentStack/Courses/Courses.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, BaseSyntheticEvent } from 'react';
+
+import Avatar from 'components/Avatar';
+import { PATHS } from 'constants/routes';
+import { NO_COURSES } from 'constants/messages';
+import { Size } from 'enums/sizes';
+import { ICoursesProps } from 'pages/SkillsMap/types';
+import transformRoute from 'utils/helpers/paths/transformRoute';
+
+import { CourseWrapper, ImageWrapper, CourseTitle, NoCoursesText } from './styled';
+
+const Courses: FC<ICoursesProps> = ({ courses }) => {
+  return (
+    <>
+      {courses.length ? (
+        <>
+          {courses.map(({ _id: courseId, avatar, title, isCompleted }) => {
+            const handleCourseClick = ({ preventDefault }: BaseSyntheticEvent) => {
+              if (isCompleted) {
+                preventDefault();
+              }
+            };
```

Comment:
Looks like logic, that should be in the container, not in the component, isn't it? :thinking:

Reply:
done